etc/rc3

```diff
@@ -30,3 +30,4 @@ flux module remove -f kvs
 flux content flush
 flux module remove -f content-sqlite
 flux module remove -f content-files
+flux module remove -f content-s3
```

Comment:
Is this a stray line left in `rc3`? (Doesn't cause any failures because of `-f`)

Reply:
I started this pattern and Garrett was continuing it, but you are right probably this should include logic similar to rc1 to figure out exactly which module to unload instead of the scattershot approach.

Reply:
Ah sorry, I didn't noce the existing `flux module remove` lines. Sorry!